CMakeLists.txt

```diff
@@ -9,6 +9,19 @@ project(Ship VERSION 8.0.6 LANGUAGES C CXX)
 include(CMake/soh-cvars.cmake)
 include(CMake/lus-cvars.cmake)
 
+option(SUPPRESS_WARNINGS "Suppress warnings in LUS and src (decomp)" ON)
+if(SUPPRESS_WARNINGS)
+    MESSAGE("Suppressing warnings in LUS and src")
+    if(MSVC)
+        set(WARNING_OVERRIDE /w STRING "")
+    else()
+        set(WARNING_OVERRIDE -w STRING "")
+    endif()
+else()
+	unset(WARNING_OVERRIDE)
```

Comment:
There is nothing to `unset` if the `set` is behind a condition, this can be dropped.